### ISSUE TITLE
quiz.php: support escape sequences

### ISSUE THREAD

**Issue by jonschipp** [jonschipp/quiz#14]
Support printf escape sequences in QA especially tab and newline for formatting.
e.g. Is this a\ttab and \nnewline?,true,false

```
          \a      Write a <bell> character.
           \b      Write a <backspace> character.
           \c      Ignore remaining characters in this string.
           \f      Write a <form-feed> character.
           \n      Write a <new-line> character.
           \r      Write a <carriage return> character.
           \t      Write a <tab> character.
           \v      Write a <vertical tab> character.
           \'      Write a <single quote> character.
           \\      Write a backslash character.
           \num
           \0num   Write an 8-bit character whose ASCII value is the 1-, 2-, or 3-digit octal number num.
```

**Comment by jonathonklem** [comment]
Most of these are already supported with fix #5

**Comment by jonschipp** [comment]
Excellent, thanks

On Mon, Mar 2, 2015 at 8:03 AM, Jonathon Klem <notifications@github.com>
wrote:

> Closed #14 <https://github.com/jonschipp/quiz/issues/14>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jonschipp/quiz/issues/14#event-244366654>.
>



-- 
Jon Schipp,
jonschipp.com, sickbits.net, opennsm.ncsa.illinois.edu
